go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cert-manager/trust-manager
 
-go 1.20
+go 1.21.1
```

Comment:
suggestion: this feels wrong? If we're giving a specific patch release it should surely be 1.21.3 (latest) - but I'd think we should just use `1.21` instead, right?

Reply:
I think you cannot set just 1.21 anymore.

Reply:
I updated it to 1.21.3